src/app/core/components/input/input.component.ts

```diff
@@ -19,25 +21,22 @@ export class InputComponent implements OnInit {
   ngOnInit(): void {
   }
 
-  onAddVideo(form: NgForm){
-    //Logic for validation returning object with api source and video ID -- WATCH OUT IF/IF/ELSE !!!
-    //And there is too much place to configure new link!!! Work on this!
+  onAddVideo(form: NgForm): void{
     if (this.userInput.validatePath(form.form.value.video)){
       let dataToFetch = {
         platform: this.userInput.extractPlatform(form.form.value.video),
         id: this.userInput.extractId(form.form.value.video)
       }
-
-      if (dataToFetch.platform == "youtube"){
-        this.youtube.fetchVideo(dataToFetch.id)
+    if (dataToFetch.platform == "youtube"){
+        this.youtube.fetchVideo(`${dataToFetch.id}`)
       }
-
-      if (dataToFetch.platform == "vimeo"){
-        this.vimeo.fetchVideo(dataToFetch.id)
+    if (dataToFetch.platform == "vimeo"){
+        this.vimeo.fetchVideo(`${dataToFetch.id}`)
       }
-
     } else {
-      console.log("Nie obsługuję tego linku lub link zawiera błędy")
+
+      console.log("Nie obsługuję tego linku lub link zawiera błędy!")
```

Comment:
tak jak wspominałem podczas calla, gdy wystawiasz pr-ka zadbaj o to aby żadnych conosle.log czy komentarzy nie było